api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -101,21 +116,22 @@ message GetPaymentStateReply {
 // - FAILED
 // - INSUFFICIENT_SIGNATURES
 enum BlobStatus {
+  // UNKNOWN means that the status of the blob is unknown.
```

Comment:
Will UNKNOWN be transitioned to a different state?

Reply:
`UNKNOWN` is a terminal status. Added to the list of terminal statuses, and also added this message:

```
  // UNKNOWN means that the status of the blob is unknown. 
  // This is a catch all and should not be encountered absent a bug.
  UNKNOWN = 0;
```